tests/test_imagecodecs.py

```diff
@@ -2930,18 +2930,20 @@ def test_ljpeg(fname, result, codec):
         decode = imagecodecs.jpeg8_decode
         check = imagecodecs.jpeg8_check
         if fname in {
-            '2ch.ljp',  # Unsupported JPEG data precision 14
             '2dht.ljp',  # Unsupported color conversion request
             'rgb24.ljp',  # Unsupported color conversion request
             'linearraw.ljp',  # Unsupported color conversion request
             'dng0.ljp',  # Invalid progressive/lossless parameters Ss=0 ...
-            # 'dng1.ljp',  # Bogus Huffman table definition
-            # 'dng2.ljp',  # Bogus Huffman table definition
-            # 'dng3.ljp',  # Bogus Huffman table definition
-            # 'dng4.ljp',  # Bogus Huffman table definition
-            # 'dng5.ljp',  # Bogus Huffman table definition
-            # 'dng6.ljp',  # Bogus Huffman table definition
-            # 'dng7.ljp',  # Bogus Huffman table definition
+            # The below "Bogus Huffman table definition" errors can be fixed
+            # with the suggested patch discussed here:
+            # https://github.com/libjpeg-turbo/libjpeg-turbo/issues/765
+            'dng1.ljp',  # Bogus Huffman table definition
```

Comment:
@cgohlke, do you know how these "dng*.ljp" image were generated (to understand why the contain the invalid Huffman codes discussed [here](https://github.com/libjpeg-turbo/libjpeg-turbo/issues/765))?

Reply:
...never mind, I know who's responsible for those images... :sweat_smile: I'll fix them.